bindings/csharp/Makefile

```diff
@@ -38,12 +38,10 @@ else
 	CKZG_LIBRARY_PATH = Ckzg.Bindings/runtimes/$(LOCATION)/native/ckzg$(EXTENSION)
 endif
 
-FIELD_ELEMENTS_PER_BLOB ?= 4096
 INCLUDE_DIRS = ../../src ../../blst/bindings
 TARGETS = ckzg.c ../../src/c_kzg_4844.c ../../blst/$(BLST_OBJ)
 
-CFLAGS += -O2 -Wall -Wextra -Wpedantic -shared
-CFLAGS += -DFIELD_ELEMENTS_PER_BLOB=$(FIELD_ELEMENTS_PER_BLOB)
+CFLAGS += -O2 -Wall -Wextra -shared
```

Comment:
No `-Wpedantic`?

Reply:
Just removed it a minute ago. See this comment:

* https://github.com/ethereum/c-kzg-4844/pull/377#issuecomment-1768406294